modules/odr_frontend/src/lib/server/pg.ts

```diff
@@ -0,0 +1,84 @@
+import { get } from 'http';
+import pg from 'pg';
+
+export interface IDBUser {
+	id: number;
+	email: string;
+	is_active: boolean | null;
+	is_superuser: boolean;
+	created_at: Date;
+	updated_at: Date | null;
+	user_type: 'user' | 'admin';
+	name: string;
+	emailVerified: boolean;
+	image: string | null;
+}
+
+export interface IDBTeam {
+	id: number;
+	name: string;
+	created_at: Date;
+	updated_at: Date | null;
+}
+
+export interface IDBUserTeam {
+	team_id: number;
+	user_id: number;
+	role: 'member' | 'admin';
+	name: string;
+}
+
+export const pg_client_config = {
+	host: process.env.POSTGRES_HOST,
+	user: process.env.POSTGRES_USER,
+	password: process.env.POSTGRES_PASSWORD,
+	database: process.env.POSTGRES_DB,
+	connectionTimeoutMillis: 2000
+};
+
+export const pgClient = new pg.Client(pg_client_config);
+
+// Connect the client once when the module is loaded
+pgClient.connect();
+
+export const PG_API = {
+	users: {
```

Comment:
We have endpoints for most of these operations on backend side.

The issue with having postgres calls on both frontend and backed is that if we find any issue with our workflow, we have to update it on both sides.

I know that you probably didn't want to wait for other team members to implement these changes which were needed for your feature and I also think we should do it the fast way to keep things running
But IMO it is something we should fix, at least in the future.